### PR TITLE
1891 real paginate ae

### PR DESCRIPTION
**Pull request by cy**

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/AssociatedEntitiesTable/AssociatedEntitiesTable.js [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+                  >
+                    {fieldToDisplay[searchField].name}
+                    <span style={{ paddingLeft: '4px' }}>
+                      <DownCaretIcon />
+                    </span>
+                  </Row>
+                }
+              >
+                {Object.keys(fieldToDisplay).map(field => (
+                  <DropdownItem
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/modern_components/AssociatedEntitiesTable/AssociatedEntitiesTable.js]: could we add `cursor: 'pointer'` here?

File: src/packages/@ncigdc/modern_components/AssociatedEntitiesTable/AssociatedEntitiesTable.js [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+                    const trimmed = trim(e.target.value);
+                    setSearchValue(trimmed);
+                    debouncedPush(searchField, trimmed, aeFilters, push);
+                  }}
+                  type="text"
+                />
+                {!!searchValue.length && (
+                  <CloseIcon
+                    style={{
+                      position: 'absolute',
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/modern_components/AssociatedEntitiesTable/AssociatedEntitiesTable.js]: same, cursor: pointer here

File: src/packages/@ncigdc/modern_components/AssociatedEntitiesTable/AssociatedEntitiesTable.js [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+                  borderBottom: `1px solid ${theme.greyScale5}`,
+                  borderRight: 0,
+                  fontSize: '14px',
+                  padding: 0,
+                }}
+                button={
+                  <Row
+                    style={{
+                      padding: '0 4px 0 4px',
+                      marginBottom: '-1px',
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/modern_components/AssociatedEntitiesTable/AssociatedEntitiesTable.js]: cursor: pointer?

**Comment by cy** [comment]
@alex-wilmer thanks, cursor pointered

**Pull request merged**